tutorial/10_key_features/004_distributed.py

```diff
@@ -1,12 +1,30 @@
 """
 .. _distributed:
 
-Distributed Optimization
-========================
+Easy Parallelization
+====================
 
-There is no complicated setup but just sharing the same study name among nodes/processes.
+It's straightforward to parallelize ``optuna.optimize()``.
 
-First, create a shared study using ``optuna create-study`` command (or using :func:`optuna.create_study` in a Python script).
+If you want to manually execute Optuna optimization:
+
+    1. create a study
+    2. share the study among multiple nodes and processes
+
+Of course, you can use Kubernetes as in `the kubernetes examples <https://github.com/optuna/optuna/tree/master/examples/kubernetes>`_.
+
+To just see how parallel optimization works in Optuna, check the below video.
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/J_aymk4YXhg?start=427" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+Create a Study
+--------------
+
+You can create a study using ``optuna create-study`` command.
+Alternatively, in Python script you can use :func:`optuna.create_study`.
```

Comment:
I found that the code example could be executed by multiple processes in Colab. I think we can make this tutorial executable, so please take a look:
https://colab.research.google.com/drive/1teCFrK_umCjW4vJeKW_1-ntJ9pKtbAOC#scrollTo=HMlDDnZK17zm

Reply:
In that case, will the generated python script be also executable as well as the generated jupyter notebook?
I'm quite not sure whether you want the tutorial to be executable in both python script and jupyter notebook or either way.

Reply:
I revisit the notebook and I don't think it's a usual usage of the jupyter notebook such as the following cell:

> `!python run.py & python run.py`

Please keep this page as it is. Sorry for bothering you.